classes/migration/upgrade/v3_4_0/UpgradeMigration.inc.php

```diff
@@ -80,6 +86,67 @@ public function up(): void
                 $table->dropColumn('setting_type');
             });
         }
+
+        // pkp/pkp-lib#6073: Delete SubmissionFiles/Notes entries that correspond to nonexistent queries.
+        $orphanedIds = DB::table('notes AS n')
+            ->leftJoin('queries AS q', 'n.assoc_id', '=', 'q.query_id')
+            ->where('n.assoc_type', '=', self::ASSOC_TYPE_QUERY)
+            ->whereNull('q.query_id')
+            ->pluck('n.note_id', 'n.assoc_id');
+
+        foreach ($orphanedIds as $neQueryId => $noteId) {
+            error_log("Removing submission files that relates to the entry ID ${noteId} which will be deleted as orphan object");
```

Comment:
Suggest "that relate to the note ID" instead of "that relates to the entry ID". "Entry ID" won't mean much.

Reply:
Changed